src/Inventory.h

```diff
@@ -35,7 +35,7 @@ class cInventory :
 	// tolua_begin
 {
 public:
-
+	int currentRecipe = 155;
```

Comment:
This line confuses the Bindings generator. I understand from the code it's only temporary and only used in a single function, so I suggest you move it as a `static` variable to that function.

Reply:
:+1: Exactly